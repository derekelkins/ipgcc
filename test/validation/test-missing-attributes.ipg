S ->
    repeat A.x
    repeat A.b until B
    { y = A.c }
    for i = 0 to 3 do A[i, i+ 1]
    { z = A(1).x };

A -> { x = . } / { y = . };

B -> "b";
