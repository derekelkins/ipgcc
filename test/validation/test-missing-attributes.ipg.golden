x isn't a guaranteed attribute on A in rule S
b isn't a guaranteed attribute on A in rule S
c isn't a guaranteed attribute on A in rule S
x isn't a guaranteed attribute on A in rule S